crates/store/re_dataframe/src/query.rs

```diff
@@ -849,24 +849,50 @@ impl QueryHandle<'_> {
                 // will not be part of the current row.
                 let mut cur_cursor_value = cur_cursor.load(Ordering::Relaxed);
 
-                // TODO(cmc): make this a tiny bit smarter so we can remove the need for
-                // deduped_latest_on_index, which would be very welcome right now given we don't
-                // have an Arrow ListView at our disposal.
-                let cur_indices = cur_chunk.iter_indices(&state.filtered_index).collect_vec();
+                let cur_index_times_empty: &[i64] = &[];
```

Comment:
I'm surprised this doesn't break any of the static related tests -- need a second to play all of that in my head